### ISSUE TITLE
[Feature Request] Replace Stereo versions with LeftRight ones (Multiband Compressor / Multiband Gate)

### ISSUE THREAD

**Issue by p-chan5** [wwmm/easyeffects#2383]
Hi developers! Currently the equalizer is the only one that can run `Split Channels` but it would be great if it was also present in these two plugins.

That would allow me to provide better worked surround sound for EasyEffects users using headphones, you can check this out in my repository: [EasyPulse](https://github.com/p-chan5/EasyPulse#sophisticated-surround-).

Thank you for your time and I hope you can make it happen :D

**Comment by wwmm** [comment]
>Thank you for your time and I hope you can make it happen :D

It should be possible. But it is hard to say when I will have time for this.

**Comment by Digitalone1** [comment]
This is a big one and preset format will change.

**Comment by p-chan5** [comment]
And if they raise it for the future? Maybe the format of the preset can be compatible as it happened with 6.3.0 (when loading it in 7.0.0).

I think it would be something that users would love since it would be something like a "dynamic surround" that is, the content would move from left to right (something like 3D panning). Something that is impossible to achieve with the Equalizer since it is static.

<img src=https://github.com/wwmm/easyeffects/assets/123143444/0782b0a6-48e6-48f8-8a80-0c515bdfa781>

**Comment by Digitalone1** [comment]
> And if they raise it for the future? Maybe the format of the preset can be compatible as it happened with 6.3.0 (when loading it in 7.0.0).

Nope, left-right plugins are different from stereo plugins. Compressors and Gates would need big refactoring to apply different configurations to a single channel. The preset format would be similar to the Equalizer, so it's incompatible with the current implementation.

> I think it would be something that users would love since it would be something like a "dynamic surround" that is, the content would move from left to right (something like 3D panning). Something that is impossible to achieve with the Equalizer since it is static.


Sure, it's a big improvement, but now moving to those implementations would require a very long work.

**Comment by p-chan5** [comment]
I understand, the code needs to be rewritten, don't worry and take your time, after all, you have already done a great (and the most difficult) job on the programming side.

If one day you decide to implement this feature, rest assured that I will also do my best so that end users can enjoy it as well <3

Thanks for your time. Regards.

**Comment by p-chan5** [comment]
So today I had some time to test this idea and I think I discovered something interesting...

It is possible to choose the Sidechain source for both (Compressor / Gate) I found it curious that it showed the left and right side. So the result is greater stereo separation coupled with a slight panning of the signal.

![0001](https://github.com/wwmm/easyeffects/assets/123143444/ba2e1c9a-77d6-4161-99ec-bf664c1cbe81)

I have to play more with this... I would like to achieve a result similar to the one in the video [What is Sony's 360 Reality Audio? Audio Samples and Stereo Comparison!](https://www.youtube.com/watch?v=ZSzLQknhU9Q)

I would like to have it ready in case one day they implement this feature. I'll be patient, I promise. It's just that I'm kind of excited to have it :D

**Comment by p-chan5** [comment]
Good news guys!

The new Stereo Split feature; added to all stereo dynamics plugins in [1.2.10](https://github.com/sadko4u/lsp-plugins/releases/tag/1.2.10), allows independent compression to be applied to the left and right channels.

So there's no need to refactor the code anymore, or undo all the great work you've done so far! It will be enough to add this option and some others from #2533

https://github.com/wwmm/easyeffects/assets/123143444/598c2ded-8176-41fb-a82d-0b3107163eaa

https://github.com/wwmm/easyeffects/assets/123143444/06d3903a-284b-460b-9cda-491e67bc1921
